### PR TITLE
Feature/passenger field

### LINKED ISSUE #233: New route - passenger field
Typling to the box should filter selector - now it doesn't provide any value.

### PR DESCRIPTION
**Pull request by pniedzwiedzinski**
I added filtering on passenger field in the new drive form (using `v-select`)

This PR closes #233

### CODE REVIEW

**Comment by pniedzwiedzinski** [comment]
Does anyone know why there are old commits showing up?

File: frontend/src/main.js [hunk 1]
```diff
@@ -1,11 +1,12 @@
 import 'bootstrap-vue/dist/bootstrap-vue.css';
 import 'bootstrap/dist/css/bootstrap.css';
-import FlagIcon from 'vue-flag-icon';
```
Comment (magul) [inline frontend/src/main.js]: Is there's some internal logic of changes in that file? (as it looks rather as import sorting and some formating)

File: frontend/src/views/DriveFormView.vue [hunk 2]
```diff
@@ -312,4 +321,28 @@ export default {
 .wrapper {
   @include m(2);
 }
+
+.select {
```
Comment (magul) [inline frontend/src/views/DriveFormView.vue]: Do we need so extensive styling?

File: frontend/src/main.js (hunk 1 above)
Comment (pniedzwiedzinski) [reply frontend/src/main.js]: Should I revert that?

Comment (magul) [reply frontend/src/main.js]: No, I'm just curious why such changes.

Comment (pniedzwiedzinski) [reply frontend/src/main.js]: I was changing there but I’ve found a simpler solution ;)

File: frontend/src/views/DriveFormView.vue (hunk 2 above)
Comment (pniedzwiedzinski) [reply frontend/src/views/DriveFormView.vue]: No ;) fixed

**Comment by arturtamborski** [comment]
Tested locally right after latest commit (fixes local storage caught by @magul) and it works fine.
Green light from my side.

**Pull request merged**